package.json

```diff
@@ -1,23 +1,44 @@
 {
-  "name": "@kaliber/library",
-  "version": "1.0.0",
+  "name": "@kaliber/sanity-plugin-duplicate",
+  "version": "0.0.1",
+  "description": "Adds a (usable) duplicate to sanity",
   "main": "index.js",
   "sideEffects": false,
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "build": "sanipack build",
+    "watch": "sanipack build --watch",
+    "verify": "sanipack verify",
+    "prepublishOnly": "sanipack build && sanipack verify",
+    "prepare": "[ -f scripts/addPeerDependencies.js ] && node scripts/addPeerDependencies.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kaliberjs/sanity-plugin-duplicate.git"
+  },
   "keywords": [
-    "hello"
+    "sanity",
+    "sanity-plugin",
+    "duplicate"
   ],
   "files": [
     "index.js",
     "src/**",
     "LICENSE",
     "readme.md",
-    "package.json"
+    "package.json",
+    "sanity.json",
+    "config.dist.json"
   ],
```

Comment:
Volgens mij zet `sanipack` dingen in een `lib` directory. Die zal je hier ook neer moeten zetten.

Reply:
Als ik lib er in moet zetten, moet  "prepublishOnly": "sanipack build && sanipack verify", dit dus wel in package.json staan? Ik denk dat ik niet helemaal de werkwijze begrijp

Reply:
Ik denk dat de verwarring was dat een andere comment over meerdere regels leek te gaan.